README.md

```diff
@@ -3,125 +3,125 @@
+                       Default: None
 
-Step 2: Get the source
-
-Windows:
-
-Step 3: At the command prompt, run
-  
-    Setup.cmd
-
-Step 4: Done! You can now use dotnet new3 in that console session
-
-Other Platforms:
-
-Step 3: In src/dotnet-new3/project.json make sure that the runtime ID for your platform is included in the runtimes section
-
-Step 4: At the command prompt, run
-  
-    dotnet restore --infer-runtimes --ignore-failed-sources
-          
-Step 5: Change the working directory to src/dotnet-new3 and run
-  
-    dotnet build -r {your RID here} -c Release
-
-Step 6: Change the working directory to src/Microsoft.TemplateEngine.Core and run
-  
-    dotnet build -c Release
-    dotnet pack -c Release -o ../../feed
-
-Step 7-10: Repeat step 6 for the following directories
-  
-    src/Microsoft.TemplateEngine.Abstractions
-    src/Microsoft.TemplateEngine.Runner
-    src/Microsoft.TemplateEngine.Orchestrator.VsTemplates
-    src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects
-
-Step 11: Prepend the following to the PATH environment variable
-  
-    {path to the source}/src/dotnet-new3/bin/Release/netcoreapp1.0/{your RID here}/
-
-Step 12: Create a directory in the root of your user profile called `.netnew`
-
-Step 13: Create a nuget.config file in the directory created in step 12 with the following contents
-  
-```xml
-<?xml version="1.0" encoding="utf-8"?>
-<configuration>
-    <packageSources>
-        <clear />
-        <!-- Add sources for templates and components here -->
-        <add key="Templates" value="{path to the source}\template_feed" />
-        <add key="Components" value="{path to the source}\feed" />
-        <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" />
-        <add key="CLI Deps" value="https://dotnet.myget.org/F/cli-deps/api/v3/index.json" />
-    </packageSources>
-    <packageRestore>
-        <add key="enabled" value="False" />
-        <add key="automatic" value="False" />
-    </packageRestore>
-</configuration>
+  -uld|--use-local-db  Whether or not to use LocalDB instead of SQLite
+                       bool - Optional
+                       Default: false
+
+  -f|--framework
+                           1.0    - Target netcoreapp1.0
+                           1.1    - Target netcoreapp1.1
+                       Default: 1.0
 ```
 
-Step 14: To make sure everything worked correctly, run
-  
-    dotnet new3 -c
+Let's create a new project named "MyAwesomeProject" in the "src/MyProject" directory. This project should be an ASP.NET MVC project with Individual Auth. To create that template
+execute `dotnet new mvc MyAwesomeProject -o src/MyProject -au Individual`. Let's try that now, the result is below.
 
-#Installing Templates
+```bash
+$ dotnet new mvc MyAwesomeProject -o src/MyProject -au Individual
+Content generation time: 302.4513 ms
+The template "MVC Application" created successfully.
+```
 
```

Comment:
This isn't a for sure thing - it was proposed & we've explored it as an option. If we were to do it, this would be defined in the template and could be controlled via a switch like `dotnet new mvc --restore-after-create`.